src/native_main.nim

```diff
@@ -114,9 +114,11 @@ proc handleMessage(msg: MessageRecv): string =
                 reply.code = some(1)
 
         of "run":
-            # this seems to use /bin/sh rather than the user's shell
-            let (shell, switch) = (if defined(windows): ("cmd", "/c") else: (getEnv("SHELL", "/bin/sh"), "-c"))
-            reply.content = some($ execProcess(shell, args=[switch, msg.command.get()], options={poStdErrToStdOut}))
+            reply.content = some $(if defined(windows):
```

Comment:
This probably needs to use `when`, not `if`, since it's at compile time.

Reply:
Huh. I tried it and it does work as it should using `if`. I tried `when` as well – and it does work –, but there's no noticeable difference in performance and and the formatting rules are stricter for some reason.

Reply:
Yeah, I noticed that too. I had to write the code to notice it so I committed it.

So `cmd /c` works as expected? You don't get command prompts popping up everywhere? (Not that I'm sure that would be a bad thing, Tridactyl users would look like real hackers)